iguana/coins/gleecbtc_7779

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Coin is named GLEEC in 3p_coins.json and seems to be referred to that in the dpow code. 

Can this file match and be named `gleec_7779` or is there another reason to keep this as `gleecbtc_7779`?

Reply:
thanks for feedback @webworker01 - renamed 7779 (chainparams) file in 69a9e4a